### ISSUE TITLE
Request to be a Developer

### ISSUE THREAD

**Issue by Cats-United** [Cats-United/Cats-Site#3]
If you want to be a developer, make sure to read the [Become a Dev](https://github.com/Cats-United/Cats-Site/wiki/Become-a-Dev) wiki page, then copy the form, enter what's in the required fields and you'll be ready!

**Comment by Firedrake969** [comment]
SQL and jQuery aren't frameworks, btw

**Comment by Cats-United** [comment]
What are they then?

**Comment by ealgase** [comment]
Username: ealgase
Age (optional): N/A
Languages I'm fluent in: HTML, CSS, Javascript
Frameworks I'm familiar with: jQuery(somewhat)

**Comment by Cats-United** [comment]
Yes! I still need someone else to do backend though.

**Comment by Firedrake969** [comment]
if you moved to heroku...

**Comment by Cats-United** [comment]
The free one only uses 512MB + it can't run 24/7 + my parents won't allow me to pay for the better things.

**Comment by Cats-United** [comment]
@ealgase - Since the backend is the main focus, you'll mainly be used for frontend and if there's Javascript in the login system (there is, but when required).

**Comment by Firedrake969** [comment]
1.  You won't use 512 mb up unless you're doing super-intensive things
2.  people won't use it 24/7

**Comment by Cats-United** [comment]
I am hosting part of it on Heroku, but only temporarily for a test of the backend.

**Comment by Firedrake969** [comment]
Use Djangooooo

**Comment by Cats-United** [comment]
I'll use Gwiddle for now unless it gets popular or something. Yeah sorry :package:

**Comment by Cats-United** [comment]
Also, what do I put in for the company name?

**Comment by Firedrake969** [comment]
Why not something more professional like x10 if you must use PHP?
Just do your username for the company name, it doesn't really matter I think

**Comment by Cats-United** [comment]
I heard that a forum was on x10 and it got removed.

**Comment by Firedrake969** [comment]
You can hear anything on the internet ;)

Ironically enough, here are some online things about PHP...
http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/
http://webonastick.com/php.html

**Comment by Cats-United** [comment]
@Firedrake969 - It happened to Opensprites.

**Comment by Firedrake969** [comment]
Well, I personally would be scared of letting a site dedicated to hosting files be using my service... it's easy to make OpenSprites sound like it's meant for doing illegal things, after all.

**Comment by Firedrake969** [comment]
>Every file on your site must have to do with the website you put up, we are not a file hosting service.

**Comment by Cats-United** [comment]
I managed to get this far (I didn't want to name it whispering-beyond-1060)
![](http://i.cubeupload.com/DFjTdn.png)

**Comment by The-Cheese-Eater** [comment]
@Cats-United Sorry to break into a conversation, but OpenSprites was fine with x10. :P The only reason we switched was due to the dodgy hosting which broke various dynamic parts of the site. ;)

**Comment by Cats-United** [comment]
Yeah, might not be using x10 for hosting @Firedrake969 .

**Comment by Cats-United** [comment]
I managed to rename the project, but that's pretty much the only thing I got done in 1.5 hours. The site still looks like the picture above.

**Comment by Firedrake969** [comment]
`git push heroku master`?

**Comment by Cats-United** [comment]
That's not my actual name :package:

**Comment by Cats-United** [comment]
I managed to get it detected as a PHP app, which I couldn't get before, but now I think I have to install Composer. I was reading about it, installed the installer and found out that it required the path for php.exe. So, I tried finding a directory called C:\PHP 5, but that doesn't come with Windows. :package: I went onto the php site to look for getting php.exe, and what I got was running PHP from the command line. I thought that php.exe was designed to be used as a Windows Web Server (which isn't a good idea, even Windows Server isn't the best server system around).

**Comment by Cats-United** [comment]
Yeah I might just go and test it on Gwiddle instead, it's far easier and i don't have to download so many things. :package:
